stack.yaml

```diff
@@ -1,6 +1,6 @@
 resolver: lts-10.3
 extra-deps:
-- time-units-1.0.0
+- o-clock-0.0.0
```

Comment:
`o-clock` actually is on Stackage. Could you please add one more `stack` file called `stack-nightly.yaml` for this resolver:

* https://www.stackage.org/nightly-2018-01-29

And you can remove `extra-deps` from this `.yaml` file because this resolver contains `universum-1.0.4` and `o-clock`. Also would be good if you add this `.yaml` file to Travis CI.